docs/conf.py

```diff
@@ -63,17 +63,17 @@
 
 # General information about the project.
 project = 'RENKU'
-copyright = '2017, Swiss Data Science Center'
-author = 'Eric Bouillet, Julien Eberle, Jiri Kuncar, Izabela Moise, Rok Roskar, Sofiane Sarni, Sandra Savchenko De Jong, Johann-Michael Thiebaut, Olivier Verscheure'
+copyright = '2017-2018, Swiss Data Science Center'
+author = 'Eric Bouillet, Julien Eberle, Jiri Kuncar, Izabela Moise, Rok Roskar, Sofiane Sarni, Sandra Savchenko De Jong, Johann-Michael Thiebaut, Fotis Georgatos, Olivier Verscheure'
```

Comment:
It was alphabetical

Reply:
doh... I'll fix it in the docs PR I have open